DFKPS/iocBoot/iocDFKPS-IOC-01/st-common.cmd

```diff
@@ -30,7 +30,7 @@ stringiftest  "SLEW"  "$(USE_SLEW=0)"  5  "1"
 epicsEnvSet "SP_PINI" "$(SP_AT_STARTUP=NO)"
 
 ## Load record instances
-dbLoadRecords("$(TOP)/db/DFKPS_common.db", "device=$(MYPVPREFIX)$(IOCNAME), P=$(MYPVPREFIX)$(IOCNAME):, FAC=$(FACTOR=1000), port=L0,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0),SP_PINI=$(SP_PINI)")
+dbLoadRecords("$(TOP)/db/DFKPS_common.db", "device=$(MYPVPREFIX)$(IOCNAME), P=$(MYPVPREFIX)$(IOCNAME):, FWI=$(FACTOR_WRITE_I=1000), FRV=$(FACTOR_READ_V=1), port=L0,RECSIM=$(RECSIM=0),DISABLE=$(DISABLE=0),SP_PINI=$(SP_PINI), VADC=$(VADC=2)")
```

Comment:
I like the use of a macro with long name that we can choose to override alter if needed without code change.